package.json

```diff
@@ -72,6 +71,7 @@
     "mocha": "^9.2.2",
     "mocha-each": "^2.0.1",
     "msw": "^0.39.2",
+    "node-fetch": "^3.2.10",
```

Comment:
You have moved `node-fetch` from dependencies to devDependencies. It won't be installed in production or when packaging (only for development). Is that intended?

Reply:
Yes. Its only used for tests.